static/js/panel.js

```diff
@@ -13,7 +13,7 @@ var Panel = new (class Panel {
 
     this.permalinkNode = this.findItem("Permalink");
     this.unpermalinkNode = this.findItem("Remove the Permalink");
-    this.logNode = this.findItem("Log");
+    this.logNode = this.findItem("Git log");
     this.rawNode = this.findItem("Raw");
```

Comment:
I think `logNode` and `rawNode` can be removed because the only uses were in the accelerator case below.  (But the (un)permalinkNode and `markdown` cases all seem to still be actively used.)